packages/eslint-config/index.js

```diff
@@ -97,13 +90,6 @@ module.exports = {
     'react/no-array-index-key': 'error',
     'react/no-danger': 'error',
     'react/no-unused-state': 'error',
-    'react/prop-types': [
-      'error',
-      {
-        ignore: ['children'],
-        skipUndeclared: true,
-      },
-    ],
```

Comment:
This was giving complaints in Gamut. We don't really need it; TypeScript does this for us. Any repo big enough to use React has since converted to TypeScript.